block/src/Pos/Block/Worker.hs

```diff
@@ -81,7 +83,7 @@ blkWorkers keepAliveTimer =
 
 informerWorker :: BlockWorkMode ctx m => (WorkerSpec m, OutSpecs)
 informerWorker =
-    onNewSlotWorker True announceBlockOuts $ \slotId _ ->
+    onNewSlotWorker def announceBlockOuts $ \slotId _ ->
```

Comment:
This is just a little fetish of mine (which I know it's debatable) but I largely prefer the use of an explicit top-level binding which alias this `def`, but it contextualises it for the task at hand. By that I mean writing this, essentially:

```haskell
defaultOnNewSlotParams :: OnNewSlotParams
defaultOnNewSlotParams = ...
```

The idea is that I can now `grep` for `defaultOnNewSlowParams`, but the same cannot be said for `def`. Granted, I can grep for `instance Default OnNewSlotParams`, but it generally adds more overhead 😁 

Nothing to stop the PR from being merged, though :)

Reply:
I tend to agree, I am not a big fan of `def` either, but for some reason I decided to use it here. I won't be lazy and will change it :)